velox/substrait/SubstraitToVeloxPlan.h

```diff
@@ -25,6 +25,35 @@ namespace facebook::velox::substrait {
 /// This class is used to convert the Substrait plan into Velox plan.
 class SubstraitVeloxPlanConverter {
  public:
+  struct SplitStats {
```

Comment:
naming: Since there are not statistics, but rather information about the split, perhaps, rename to SplitInfo.